modules/ROOT/pages/rtfctl-permissions.adoc

```diff
@@ -0,0 +1,112 @@
+= RTFCTL Kubernetes Permissions
+
+Understanding and enabling the specific permissions required over the Kubernetes resources is crucial for successfully running `rtfctl` commands. Properly configured role permissions are essential for seamless execution of `rtfctl` operations.
+
+The following table lists all the permissions that you configure using K8s RBACs (https://kubernetes.io/docs/reference/access-authn-authz/rbac/[Role based access control^]) in Kubernetes: 
```

Comment:
```suggestion
The following table lists all the permissions that you configure using K8s RBACs (https://kubernetes.io/docs/reference/access-authn-authz/rbac/[role-based access control^]) in Kubernetes: 
```

Reply:
For inline links, use lower-case.

Reply:
If you're using K8s for Kubernetes, suggest defining it first.

Reply:
The intro to the table should come immediately before the table. As it reads now, this intro applies to the bullet list rather than the table.